deps/test-old.txt

```diff
@@ -5,59 +5,59 @@
-blinker==1.8.2            # via flask
+black==24.10.0            # via hypofuzz (setup.py), hypothesis
+blinker==1.9.0            # via flask
 certifi==2024.8.30        # via requests
-charset-normalizer==3.3.2  # via requests
+charset-normalizer==3.4.0  # via requests
 click==8.1.7              # via black, flask, hypothesis
-coverage[toml]==7.6.1     # via hypofuzz (setup.py), pytest-cov
-dash==2.18.1              # via hypofuzz (setup.py)
+coverage[toml]==7.6.8     # via hypofuzz (setup.py), pytest-cov
+dash==2.18.2              # via hypofuzz (setup.py)
 dash-core-components==2.0.0  # via dash
 dash-html-components==2.0.0  # via dash
 dash-table==5.0.0         # via dash
 exceptiongroup==1.2.2     # via hypothesis, pytest
 execnet==2.1.1            # via pytest-xdist
 flask==3.0.3              # via dash
-hypothesis[cli]==6.112.1  # via hypofuzz (setup.py)
+hypothesis[cli]==6.121.0  # via hypofuzz (setup.py)
 idna==3.10                # via requests
 importlib-metadata==8.5.0  # via dash
 iniconfig==2.0.0          # via pytest
 itsdangerous==2.2.0       # via flask
 jinja2==3.1.4             # via flask
-libcst==1.4.0             # via hypofuzz (setup.py)
+libcst==1.5.1             # via hypofuzz (setup.py)
 markdown-it-py==3.0.0     # via rich
-markupsafe==2.1.5         # via jinja2, werkzeug
+markupsafe==3.0.2         # via jinja2, werkzeug
 mdurl==0.1.2              # via markdown-it-py
 mypy-extensions==1.0.0    # via black
 nest-asyncio==1.6.0       # via dash
-numpy==2.0.2              # via -r deps/./test.in, pandas, pyarrow
-packaging==24.1           # via black, plotly, pytest
+numpy==2.0.2              # via -r /Users/tybug/Desktop/Liam/coding/hypofuzz/deps/test.in, pandas
+packaging==24.2           # via black, plotly, pytest
 pandas==2.2.3             # via hypofuzz (setup.py)
 pathspec==0.12.1          # via black
 platformdirs==4.3.6       # via black
 plotly==5.24.1            # via dash
 pluggy==1.5.0             # via pytest
-psutil==6.0.0             # via hypofuzz (setup.py)
-pyarrow==17.0.0           # via -r deps/./test.in
+psutil==6.1.0             # via hypofuzz (setup.py)
+pyarrow==18.1.0           # via -r /Users/tybug/Desktop/Liam/coding/hypofuzz/deps/test.in
 pygments==2.18.0          # via rich
-pytest==7.4.4             # via -r deps/./test.in, -r deps/test-old.in, hypofuzz (setup.py), pytest-cov, pytest-xdist
-pytest-cov==5.0.0         # via -r deps/./test.in
-pytest-xdist==3.6.1       # via -r deps/./test.in
-python-dateutil @ git+https://github.com/dateutil/dateutil.git@296d419fe6bf3b22897f8f210735ac9c4e1cb796  # via -r deps/./test.in, pandas
+pytest==7.4.4             # via -r /Users/tybug/Desktop/Liam/coding/hypofuzz/deps/test.in, -r deps/test-old.in, hypofuzz (setup.py), pytest-cov, pytest-xdist
+pytest-cov==6.0.0         # via -r /Users/tybug/Desktop/Liam/coding/hypofuzz/deps/test.in
+pytest-xdist==3.6.1       # via -r /Users/tybug/Desktop/Liam/coding/hypofuzz/deps/test.in
```

Comment:
is there a way to get tox to use a relative path? I guess this is the same thing we ran into in https://github.com/HypothesisWorks/hypothesis/pull/4178#discussion_r1845263735 huh?

Reply:
Haven't looked into it on either end yet; probably the quickest way forward is to do some `sed` on the files after writing.